src/store.js

```diff
@@ -10,7 +10,7 @@ const persistConfig = {
     storage: storage,
     // autoMergeLevel1 is the default anyway, but I want to be explicit
     stateReconciler: autoMergeLevel1,
-    whitelist: ['auth', 'map']
+    whitelist: ['auth', 'map', 'addSite']
```

Comment:
`redux-persist` is so freaking cool. This one-word addition automatically gets us offline persistence for the entire `addSite` set of state.

Reply:
<3